MediaBrowser.Controller/Entities/TV/Series.cs

```diff
@@ -34,6 +34,8 @@ public Series()
 
         public string AirTime { get; set; }
 
+        public Dictionary<int, string> SeasonNames { get; set; }
```

Comment:
I'm still not sold on this

Reply:
I don't like it either but haven't found a better solution for it. Additional input is always welcome 😀

Reply:
Would this need to be saved to the database too?

Reply:
Friendly bump, what can we do to make this work @joshuaboniface @cvium ?